test/test_integration.py

```diff
@@ -640,7 +640,11 @@ def on_click(n_clicks):
                             'x': [1, 2, 3, 4],
                             'y': [4, 3, 2, 1]
                         }
-                    ]
+                    ],
+                    'layout': {
+                        'width': 700,
+                        'height': 450
+                    }
```

Comment:
I'm not sure what to say about this one... I'm tempted to consider it a bug, but I don't know what we'd do to fix it. The issue - and I still don't understand why it's triggered intermittently here, but it's a real issue users will face - is that if you have an autosized graph that's initially drawn in a `display: none` container, it gets the default size. It keeps that default size when the container is displayed, because no redraw is triggered. But if the window is resized at all, THEN the graph gets sized correctly.

This seems to sometimes happen on CI - see eg https://percy.io/plotly/dash-core-components/builds/1576352?utm_campaign=plotly&utm_content=dash-core-components&utm_source=github_status - so I'm just locking down the size here.